src/blockedIps.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I add this to fallback for all the account that does not have a blocklist. Should not affect, but let me know if you think is not a good pattern, I will think about try/catching the import instead.

Reply:
Seems reasonable to me, especially as this should be short-term.